app/models/post.rb

```diff
@@ -2,16 +2,15 @@ class Post < ActiveRecord::Base
   validates_presence_of :title
   validates_presence_of :body
 
-  after_save :enqueue_create_or_update_document_job
-  after_destroy :enqueue_delete_document_job
+  after_save :create_swiftype_document
+  before_destroy :destroy_swiftype_document
```

Comment:
I think you can change this back to after_destroy now that you have it working with the Post ID.

The reason why you would want to do that is that destroys can be canceled by other callbacks! And if that happened, the Post would be deleted from Swiftype, but not from the application's database.